src/libs/brands.tsx

```diff
@@ -406,7 +403,6 @@ export const brands: Record<string, BrandConfiguration> = {
     welcomeHeader: <ScientificServicesWelcomeHeader />,
     description: <ScientificServicesDescription />,
     hostName: 'services.terra.bio',
-    compactTopBar: true,
```

Comment:
in hindsight, "compactTopBar" was a pointless and redundant concept that i introduced to the brand configurations back in the original feature branch. keying off the enabledBrand itself seems to be more in line with the other brand conditionals already used throughout terra-ui